### PR TITLE
[10] Create & Style the Customer Signup Page using CSS

### PR DESCRIPTION
**Pull request by Oyeteju**
## Task: `html and styles (css) of signup-page`
- `The page is responsive, user can view on tablet and mobile phone`
image
image
image

### CODE REVIEW

**Comment by adaugochi** [comment]
Well done @Oyeteju 
But your `input` field isn't styled exactly the way it is on the mockup

File: submissions/oyetejuOloyede/css/signUp.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/oyetejuOloyede/css/signUp.html:1]: @Oyeteju Move this file to your htmlForms folder.

File: submissions/oyetejuOloyede/css/style.css [hunk 2]
```diff
@@ -0,0 +1,199 @@
+*{
+    box-sizing:border-box;
+}
+
+body{
+    font: normal 16px Arial, Helvetica, sans-serif;
```
Comment (ToluAkin) [inline submissions/oyetejuOloyede/css/style.css:6]: You should adjust this to `font-size` and `font-family`

**Comment by ToluAkin** [comment]
@Oyeteju The form is not totally similar to the design provided. Adjust views provided and change the screenshots.

**Comment by Oyeteju** [comment]
> Well done @Oyeteju
> But your `input` field isn't styled exactly the way it is on the mockup

input field has been updated

File: submissions/oyetejuOloyede/css/signUp.html (hunk 1 above)
Comment (Oyeteju) [reply submissions/oyetejuOloyede/css/signUp.html:1]: filed moved to htmlForms folder

File: submissions/oyetejuOloyede/css/style.css (hunk 2 above)
Comment (Oyeteju) [reply submissions/oyetejuOloyede/css/style.css:6]: font and font-family adjusted

**Comment by ToluAkin** [comment]
> > Well done @Oyeteju
> > But your `input` field isn't styled exactly the way it is on the mockup
> 
> input field has been updated

@Oyeteju Add the new screenshot here

**Pull request merged**